test/phoenix/digester_test.exs

```diff
@@ -0,0 +1,34 @@
+defmodule Phoenix.DigesterTest do
+  use ExUnit.Case, async: true
+
+  test "fails when the given paths are invalid" do
+    assert {:error, :invalid_path} = Phoenix.Digester.compile("../some invalid?/path", "/other_invalid ?? /path")
+  end
+
+  test "digests and compress files" do
+    output_path = Path.join("tmp", "digested")
+    input_path = "priv/static"
+    File.mkdir_p!(output_path)
+
+    assert :ok = Phoenix.Digester.compile(input_path, output_path)
+
+    input_files = assets_files(input_path)
```

Comment:
I don't like tests like this because we are not really testing anything. If `priv/static` becomes empty (no assets) the test would still pass. I would explicitly pick one file from `priv/static` (or even create a dummy input directory in tmp) and explicitly assert on it.